integreat_cms/cms/templates/dashboard/_todo_dashboard_widget.html

```diff
@@ -13,7 +13,7 @@
                 Every day you can find a list of ideas on how you can improve the content of your pages.
             {% endblocktranslate %}
         </p>
-        {% if perms.cms.change_page %}
+        {% if perms.cms.change_page and request.region.default_language %}
```

Comment:
What do you think about moving this condition [here](https://github.com/digitalfabrik/integreat-cms/blob/4d7020f07aa5952b96de612522baa795b973c59f/integreat_cms/cms/templates/dashboard/dashboard.html#L13) as @seluianova suggested?